duecredit/cmdline/cmd_summary.py

```diff
@@ -20,10 +22,13 @@
 
 __docformat__ = 'restructuredtext'
 
+if TYPE_CHECKING:
+    import argparse
```

Comment:
There's no downside to importing `argparse`.  Why gate it behind `if TYPE_CHECKING`?